Userland/Libraries/LibWeb/CSS/Default.css

```diff
@@ -44,7 +44,7 @@ textarea {
     height: attr(rows lh, 2lh);
 }
 
-input[type=submit], input[type=button], input[type=reset], input[type=checkbox], input[type=radio] {
+input[type=submit], input[type=button], input[type=reset], input[type=color], input[type=checkbox], input[type=radio] {
```

Comment:
You don't have to do this, it's a bit of a tangent, but: This `input[type=foo], ... { }` block is just here to undo some properties set by the `input, textarea { }` block above. We can instead make that earlier block not include these by changing its selector to:

```css
input:not([type=submit], [type=button], [type=reset], [type=checkbox], [type=radio]),
textarea {
...
}
```
and then removing this second block.

Reply:
I'll correct it to that, however it will still need to exclude the color type to prevent the bug with it having an extra border and being too wide